data/definitions/npcs.yml

```diff
@@ -72,122 +33,60 @@ chicken:
   hitpoints: 30
   slayer_xp: 3.0
   max_hit_melee: 1
-  style: "stab"
-  examine: "Yep. Definitely a chicken."
-  race: "bird"
-cow_brown:
-  id: 12363
-  copy: cow
-cow_light:
-  id: 12365
-  copy: cow
-cow_calf:
-  id: 12364
+  style: stab
+  examine: Yep. Definitely a chicken.
+  race: bird
+sheep:
+  id: 43
   att: 1
   str: 1
   def: 1
   mage: 1
   range: 1
-  stab_def: -26
-  att_bonus: -20
-  str_bonus: -20
-  slash_def: -26
-  crush_def: -26
-  magic_def: -26
-  range_def: -26
+  stab_def: -10
+  att_bonus: -10
+  str_bonus: -10
+  slash_def: -10
+  crush_def: -10
+  magic_def: -10
+  range_def: -10
```

Comment:
Sheep aren't attackable so don't need combat stats 🐑

Reply:
looking at the original it was already there

Reply:
i've gone through npcs.yml should be fixed